ncm-ceph/src/main/pan/components/ceph/v1/schema-mds.pan

```diff
@@ -1,4 +1,4 @@
-declaration template components/${project.artifactId}/schema-mds;
+declaration template components/${project.artifactId}/v1/schema-mds;
```

Comment:
v1 -> Jewel

Reply:
My idea is that version of the schemas shouldnt (always) follow component code versions. While i am not planning on redoing the schema again, it is possible we want to use subclasses for feature or command changes in further releases.